_opcodes/op025.html

```diff
@@ -18,6 +19,14 @@
 <br/>3   Inflict 1HP every 'Damage Amount' seconds
 <br/>4   Inflict Param 3 Damage every Param 1 seconds
 <br/>5+  Crash
+<br/><br/>
+This opcode uses its special field as a 'Frequency Multiplier', with a default(0) value of '1'. In particular:
+<ul>
+	<li>if 'Type' is either 0 or 1, this field is unused</li>
+	<li>if 'Type' is 2, then Frequency = 'Frequency Multiplier'</li>
+	<li>if 'Type' is 3, then Frequency = 'Damage Amount' * 'Frequency Multiplier'</li>
+	<li>if 'Type' is 4, then Frequency = 'Damage Amount' * 'Frequency Multiplier'</li>
+</ul>
```

Comment:
Are you sure this isn't limited to EEs? Sounds new.

Reply:
Plus, it's probably param 3 * fm for type==4, to match what that does and to make it different to type==3.

Reply:
@lynxlynxlynx 

Updated with the latest v2.5 changes. Do you think I should put also the following?
<details>
<summary>Spoiler</summary>

If `Parameter2 = 1`
- If the `Parameter4` field is non-zero, damage is always dealt.
- If `Parameter1` is less than 256:
Damage is dealt while Current HP > 100 / `Parameter1`.
- Otherwise damage is dealt between certain thresholds:
[000 / ('Parameter1' / 65536)] < Current HP < [050 / ('Parameter1' / 65536)]
[100 / ('Parameter1' / 65536)] < Current HP < [150 / ('Parameter1' / 65536)]
[200 / ('Parameter1' / 65536)] < Current HP < [250 / ('Parameter1' / 65536)]
[300 / ('Parameter1' / 65536)] < Current HP < [350 / ('Parameter1' / 65536)]
[400 / ('Parameter1' / 65536)] < Current HP < [450 / ('Parameter1' / 65536)]
etc... up to characters maximum hit points.
</details>

Reply:
Uff, what a mess! I don't know how to read the third bullet point — if the equations are right, most values between 255 and 65536 will end up in the first category. I think it would be clearer if you added an example (how it works and why one'd want to use it).

Reply:
> Uff, what a mess! I don't know how to read the third bullet point — if the equations are right, most values between 255 and 65536 will end up in the first category. I think it would be clearer if you added an example (how it works and why one'd want to use it).

OK, I got an answer, here you go.
I don't really think there is any reason to want to use p2=1 with p1 values > 255, or even >100.
If 'Parameter1' is 65536, it will deal damage while the target's current HP is between 0 and 50, 100 and 150, 200 and 250, etc...
If 'Parameter1' is 131072, it will deal damage while the target's current HP is between 0 and 25, 50 and 75, 100 and 125, etc...
Both at a rate of 1/s. Dropping out of those ranges does not end the effect, damage will resume if they are later healed/damaged back into one of those ranges.

This function (param2=1) was intended to damage a target for a total given percentage of maximum health (See description on Arrows of Biting), but has never worked that way, and is why its mechanics are so weird.
The arrows, in their current form, damage a target until their current HP < 3.33 (100 / 30) , or their duration expires.

Reply:
Heh, that's even worse. I suggest you include just the info up to the last bullet point and leave it at that.

Reply:
> Heh, that's even worse. I suggest you include just the info up to the last bullet point and leave it at that.

Done.

On top of that, I decided to create a new HTM file so as to properly handle the EE behavior. I did the same for opcode #78, #98 and #272.